registry/event/service_instances_changed_listener_impl.go

```diff
@@ -130,12 +125,15 @@ func (lstn *ServiceInstancesChangedListenerImpl) OnEvent(e observer.Event) error
 
 		for key, notifyListener := range lstn.listeners {
 			urls := lstn.serviceUrls[key]
+			events := make([]*registry.ServiceEvent, len(urls))
```

Comment:
should use make([]*registry.ServiceEvent, 0 , len(urls))

Reply:
updated.